### ISSUE TITLE
How to get pretrained weights of  yolo-nas on obj365 datasets

### ISSUE THREAD

**Issue by Yi-dashi** [Deci-AI/super-gradients#1143]
### 💡 Your Question

How to get pretrained weights of  yolo-nas_s on obj365 datasets? yolo-nas got a mAP of 47.5, is this the result of pre-training on the obj365 dataset and then fine-tuning on the coco dataset?

**Comment by BloodAxe** [comment]
Objects365 was part of pretraining of Yolo-NAS. However we do not provide pretrained weights for Objects365, since there wasn't any goal to reach best possible accuracy on that dataset.
So Yolo-NAS weights for Objects365 will not be released.

**Comment by PavanNettam** [comment]
Are pre-trained weights of yolo-nas_s on coco (or any other dataset) released?
